layer-publisher/src/layer-publisher-stack.ts

```diff
@@ -0,0 +1,44 @@
+/* eslint-disable @typescript-eslint/explicit-member-accessibility */
+import { CfnOutput, RemovalPolicy, Stack, StackProps } from 'aws-cdk-lib';
+import { Construct } from 'constructs';
+import * as lambda from 'aws-cdk-lib/aws-lambda';
+import { LambdaPowertoolsLayer } from 'cdk-lambda-powertools-python-layer';
+import { StringParameter } from 'aws-cdk-lib/aws-ssm';
+import { CfnLayerVersionPermission } from 'aws-cdk-lib/aws-lambda';
+
+export interface LayerPublisherStackProps extends StackProps {
+  readonly layerName?: string
+  readonly powerToolsPackageVersion?: string
+  readonly ssmParameterLayerArn: string
+}
+
+export class LayerPublisherStack extends Stack {
+  readonly lambdaLayerVersion: lambda.LayerVersion;
+  constructor(scope: Construct, id: string, props: LayerPublisherStackProps) {
+    super(scope, id, props);
+
+    this.lambdaLayerVersion = new LambdaPowertoolsLayer(this, 'LambdaPowertoolsLayer', {
+      layerVersionName: props?.layerName,
+      runtimeFamily: lambda.RuntimeFamily.NODEJS,
+      version: props?.powerToolsPackageVersion,
+    });
```

Comment:
Q: Assuming that we upgrade the version from 1.0 to 1.1. I assume that this construct will just create a new Lambda Layer version?  

What if we want to rollback?  Will it delete the latest Lambda layer version?

I'm assuming the worst case and we need to do an emergency fix. Might worth documenting the rollback process in the README?